bin/nib

```diff
@@ -126,25 +155,22 @@ if __name__ == '__main__':
             print('Done')
 
         elif options.action == 'wizard':
-            empty = True
-            for dirpath, dirs, files in os.walk('.'):
-                if len(dirs) > 0 or len(files) > 0:
-                    empty = False
-                    break
-
-            overwrite = input('This directory is not empty, the wizard will overwrite existing files. Proceed? [y/N]: ')
-            if not overwrite.lower().startswith('y'):
-                sys.exit(1)
+            # If the directory's not empty
+            if os.listdir('.') != []:
+                answer = getYN( 'This directory is not empty, '
+                                'the wizard will overwrite existing files.'
+                                ' Proceed? ', False)
```

Comment:
Would prefer to drop the leading space between `getYN(` and the string literal.  Also would like to have the space before  "Proceed" moved to the previous line if possible, and drop the space after the ? as one will get printed by `getYN()`.